Parser/pegen/peg_api.c

```diff
@@ -0,0 +1,28 @@
+#include <pegen_interface.h>
+
+#include "../tokenizer.h"
+#include "pegen.h"
+
+mod_ty
+PyPegen_ASTFromString(const char *str, PyArena *arena)
+{
+    return run_parser_from_string(str, parse_start, RAW_AST_OBJECT, arena);
```

Comment:
Hm, it's pretty terrible to have `run_parser_from_string()` return sometimes a `mod_ty` and sometimes a `PyObject *`. But I think you know that and this is a temporary hack?

Reply:
See below.